airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -730,10 +718,10 @@ def is_job_dataflow_running(self, name: str, variables: Dict, project_id: Option
     @GoogleBaseHook.fallback_to_default_project_id
     def cancel_job(
         self,
+        project_id: str,
```

Comment:
Same here -> I think it needs UPDATING.md entry

Reply:
It is the only keyword argument method.